ledger/ledger-api-test-tool/BUILD.bazel

```diff
@@ -105,6 +80,70 @@ da_scala_binary(
     ],
 )
 
+tests = [(
+    path.replace("/", "_").replace(".", "_"),
+    path,
+) for path in glob([
+    "src/main/scala/com/daml/ledger/api/testtool/tests/**/*.scala",
+])]
+
+[
+    da_scala_library(
+        name = name,
+        srcs = [path],
+        deps = [
+            ":PackageManagementTest.scala",
+            ":ledger-api-test-tool-lib",
+            "//daml-lf/archive:daml_lf_archive_reader",
+            "//daml-lf/archive:daml_lf_dev_archive_java_proto",
+            "//daml-lf/data",
+            "//daml-lf/engine",
+            "//daml-lf/interpreter",
+            "//daml-lf/language",
+            "//daml-lf/transaction",
+            "//language-support/scala/bindings",
+            "//ledger/ledger-api-common",
+            "//ledger/ledger-api-domain",
+            "//ledger/ledger-api-grpc-utils",
+            "//ledger/test-common",
+            "//ledger/test-common:SemanticTests.scala",
+            "//ledger/test-common:Test-dev.scala",
+            "//ledger/test-common:Test-stable.scala",
+            "@maven//:ai_x_diff_2_12",
+            "@maven//:com_chuusai_shapeless_2_12",
+            "@maven//:org_cvogt_scala_extensions_2_12",
+        ],
+    )
+    for name, path in tests
+]
```

Comment:
As I understand, `rules_scala` defines [`scala_library_suite`](https://github.com/bazelbuild/rules_scala/blob/master/docs/scala_library_suite.md) for this purpose. You could add a `da_scala_library_suite` wrapper to `bazel_tools/scala.bzl` similar to `da_scala_test_suite` to use it here.

Reply:
Ah, very nice, thanks! I'm working on that.

Reply:
Should be mostly addressed by https://github.com/digital-asset/daml/pull/3486/commits/eb01efc7155555785b13a71e360ea412bdd2fd57 but I would greatly appreciate any feedback on that, I mostly improvised when it came to the wrapping macro.

Reply:
FYI @SamirTalwar.

Reply:
![Smort.](https://media.giphy.com/media/hFROvOhBPQVRm/giphy.gif)